SimCalorimetry/Configuration/python/SimCalorimetry_EventContent_cff.py

```diff
@@ -49,3 +57,8 @@
 from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
 phase2_common.toModify( SimCalorimetryFEVTDEBUG.outputCommands, func=lambda outputCommands: outputCommands.append('keep *_simEcalUnsuppressedDigis_*_*') )
 phase2_common.toModify( SimCalorimetryRAW.outputCommands, func=lambda outputCommands: outputCommands.append('keep *_simEcalUnsuppressedDigis_*_*') )
+
+from Configuration.Eras.Modifier_phase2_ecal_cff import phase2_ecal
+phase2_ecal.toModify( SimCalorimetryPREMIX.outputCommands, func=lambda outputCommands: outputCommands.append('drop ESDigiCollection_simEcalUnsuppressedDigis_*_*') )
```

Comment:
The convention is to use `phase2_common` for preshower removal

Reply:
Thanks, fixed.